tests/test_logging_config.py

```diff
@@ -259,6 +259,11 @@ def test_loading_remote_logging_with_wasb_handler(self):
         conf.set('core', 'remote_base_log_folder', 'wasb://some-folder')
         configure_logging()
 
+        import logging
+        from airflow.utils.log.wasb_task_handler import WasbTaskHandler
+        logger = logging.getLogger('airflow.task')
+        self.assertIsInstance(logger.handlers[0], WasbTaskHandler)
```

Comment:
Failing with:

`   AssertionError: <airflow.utils.log.file_task_handler.FileTaskHandler object at 0x7f5bf7d1a950> is not an instance of <class 'airflow.utils.log.wasb_task_handler.WasbTaskHandler'>`

Because of how configure logging works we will need to reload  before calling configure_ logging() airflow.config_templates.airflow_local_settings:

```
from six.moves import reload_module
import airflow.config_templates.airflow_local_settings
reload_module(airflow.config_templates.airflow_local_settings)
configure_logging()
```